### ISSUE TITLE
Error in tutorial steps

### ISSUE THREAD

**Issue by Antelope-IT** [aspnet/Docs#6817]
I followed the tutorial above for Visual Studio Code. At step 3 of "Create an ASPNET core project..." it instructs you to copy the file SignalR.js to the lib directory of the project however the code that's copied in step 1 of "Create the SignalR client..." is expecting the signalr.js file to be in a subfolder "lib/signalr" 
`&lt;script src="~/lib/signalr/signalr.js"&gt;&lt;/script&gt;`
The sample fails with an error until either the file is moved into the subfolder or the code is corrected so that it loads the file from the root of lib.  

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 778aaeae-2fa8-fb73-b644-8e99ee06398e
* Version Independent ID: 159a95c8-7afd-bb03-fdfc-2323559ef50a
* Content: [Get started with SignalR on ASP.NET Core](https://docs.microsoft.com/en-us/aspnet/core/signalr/get-started?view=aspnetcore-2.1&tabs=visual-studio-code)
* Content Source: [aspnetcore/signalr/get-started.md](https://github.com/aspnet/Docs/blob/master/aspnetcore/signalr/get-started.md)
* Product: **aspnet-core**
* GitHub Login: @rachelappel
* Microsoft Alias: **rachelap**

**Comment by rachelappel** [comment]
This may be a sync issue where the content was updated but not yet published. The text for Step 3 has been updated recently and states the following:
"Create a new folder named "signalr" inside the lib folder in your project. Then copy the signalr.js file from node_modules\@aspnet\signalr\dist\browser to this folder."
Step 1 is "Replace the content in Pages\Index.cshtml with the following code:". The script reference in the code is this:  "<script src="~/lib/signalr/signalr.js"></script>"

**Comment by Antelope-IT** [comment]
Hi I'm not entirely sure but I think it may have been changed for step 4 of the Visual Studio instructions, but Step 3 of the Visual Studio Code instructions doesn't mention creating a folder at all. I've looked at the source in master on GitHub and the instructions there for Visual Studio Code, as of the time of writing, do not mention creating a sub-folder. There was a change made for this issue https://github.com/aspnet/Docs/pull/6782  which sounds like it could be the one,  but looking at the changes for this https://github.com/aspnet/Docs/pull/6782/files/d3fe1364da2ce741e4e646cfc081d427c3003320 it only shows changes for the Visual Studio instructions not the Visual Studio Code ones and when I look at the rendered page above I can see the Visual Studio changes.  So could it be that the change for VS Code is in different commit or branch?
